src/src_V2/modules/manager/user-management/UserConfig/UserConfig.js

```diff
@@ -0,0 +1,225 @@
+import { useState } from 'react';
+import { useFormik } from 'formik';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Checkbox, Grid, TextField, Typography } from '@mui/material';
+
+import DeactivateDialog from '../DeactivateDialog/DeactivateDialog';
+import EditUserActions from '../../../../components/editUserActions';
+
+import {
+  resources,
+  actionLevels,
+  mockUserManagementData,
+  mockResourceDescription,
+} from '../mock/data';
+import styles from './styles.module.css';
+import {
+  userConfigInitialValues,
+  userConfigValidationSchema,
+} from '../../../../utils/schemas/userManagement/userManagement';
+
+export default function UserConfigContent({ newUser }) {
+  const userId = newUser ? null : +useParams().userId;
+  const userData = newUser
+    ? {}
+    : mockUserManagementData.find((user) => user.id === userId);
+  const navigate = useNavigate();
+
+  const submitHandler = (values) => {
+    newUser
+      ? navigate('/manager/user-management')
+      : values.status === 'Inactive'
+      ? setFieldValue('status', 'active or pending')
+      : console.log(values);
+  };
+
+  const {
+    dirty,
+    values,
+    errors,
+    touched,
+    resetForm,
+    handleSubmit,
+    getFieldProps,
+    setFieldValue,
+  } = useFormik({
+    initialValues: userConfigInitialValues(
+      newUser
+        ? userData
```

Comment:
what is the use of ternary operator there?

Reply:
user data can have a lot of unnecessary properties, so we take what is needed if we don't add new user